blocks/header/header.js

```diff
@@ -239,7 +97,7 @@ const navDecorators = { 'nav-top': decorateTopNav, 'nav-middle': decorateMiddleN
 export default async function decorate(block) {
   // fetch nav content
   const navMeta = getMetadata('nav');
-  const navPath = navMeta || (getLanguage() === 'en' ? '/nav' : `/${getLanguage()}/nav`);
+  const navPath = navMeta || (getLanguage() === 'jp' ? '/nav' : `/${getLanguage()}/nav`);
```

Comment:
This won't work as the jp version is the default for the website, but the getLanguage() function defaults to en if there's not language specified in the path

Reply:
This should work now, as I merged the Footer changes in.